docs/authentication/embedded-sdk-add-otp-email.mdx

```diff
@@ -18,11 +18,106 @@ doc_type: quickstart
 displayed_sidebar: mainSidebar
 ---
 
-import { Alert, Col, Row } from "antd";
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
 
-<Row>
-  <Col span={6}>
-    <Alert message="Coming soon!" type="warning" />
-  </Col>
-</Row>
-<br />
+This guide describes how to add email based OTP (one-time password) to your app. OTP is an auto generated code that is sent to the user's email address that can only be used once for a single login. This can be used to prove that the user has access to the email address associated with the account in question. You will find that OTP email verification is used on the [Hosted Web Authenticator](/docs/next/authentication#hosted-web) for sign-up and new browser logins. This guide will help you implement your own OTP solution for your application using the Beyond Identity Embedded SDK.
+
+## Prerequisites
+
+- [x] A [developer account](https://www.beyondidentity.com/developers)
+- [x] An [application](add-an-application) in your admin console
+- [x] Set up the [Beyond Identity Embedded SDK](sdk-setup)
+
+## Add OTP to your app
+
+Adding email based OTP to your app involves calling the following functions from the Beyond Identity Embedded SDK.
+
+Use the following guides for language specific information:
+
+- [authenticateOtp](/docs/next/js-reference-authenticateOtp)
+- [redeemOtp](/docs/next/js-reference-redeemOtp)
+
+### 1. Send the user an OTP
+
+In your application, create a text field for the user to enter an email address. After the user enters an email address you will want to call the Embedded SDK function [authenticateOtp](/docs/next/js-reference-authenticateOtp) with the current authenticate url and the user's email address. Calling this functions will enable Beyond Identity to send the user an email with a one-time password.
+
+The result of calling this function will return an [OtpChallengeResponse](/docs/next/js-reference-authenticateOtp#returns) that contains a `URL`. This URL should be saved and used when calling [redeemOtp](/docs/next/js-reference-redeemOtp) or other calls to [authenticate](/docs/next/js-reference-authenticate) or [authenticateOtp](/docs/next/js-reference-authenticateOtp) in the same transaction.
+
+```javascript
```

Comment:
Yea I was thinking since we are releasing so soon, it's not worth mentioning. It's also already mentioned on the page when you tap the link to navigate to the function.

Reply:
Yeah, that makes sense to me